Task_13/style.css

```diff
@@ -0,0 +1,45 @@
+.conteiner{
+    display: flex;
+    justify-content: center;
+    height: 50px;
+    width: 100px;
+    margin: 0 auto;
+    position: relative;
+}
+.ball{
+    background-color: #0e5a8b;
+    border-radius: 50%;
+    width: 17px;
+    height: 17px;
+    margin: 0 3px;
+    position: relative;
+    top: 5px;
+    animation-name: ball-jump;
+    animation-duration: 1s;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+    animation-direction: alternate;
+}
+.ball_1{
+    animation-delay: 0.2s;
+}
+.ball_2{
+    animation-delay: 0.4s;
+}
+@keyframes ball-jump {
+    0%{
+        top: 5px;
+        width: 17px;
```

Comment:
Oh no, too difficult for browser. (performance --).

You should have used transform for this animation.

For animations recommended to use only transform and opacity properties.